Assertion error on test; site-packages vs src

**Describe the bug**
When running tests I get the following error:

```
Run coverage run -m unittest discover && coverage report
.............F
======================================================================
FAIL: test_handle_response (tests.sharepoint.test_client.TestSharepointClient)
Tests that the responses returned are what's expected.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/aind-metadata-service/aind-metadata-service/tests/sharepoint/test_client.py", line 290, in test_handle_response
    self.assertEqual(expected_msg1.body, msg1.body)
AssertionError: b'{"m[53[8](https://github.com/AllenNeuralDynamics/aind-metadata-service/actions/runs/3743887327/jobs/6356592666#step:6:9) chars]ain/site-packages/aind_data_schema/procedures.[[9](https://github.com/AllenNeuralDynamics/aind-metadata-service/actions/runs/3743887327/jobs/6356592666#step:6:10)51 chars]ll}}' != b'{"m[538 chars]ain/src/aind_data_schema/procedures.py","schem[941 chars]ll}}'

----------------------------------------------------------------------
Ran [14](https://github.com/AllenNeuralDynamics/aind-metadata-service/actions/runs/3743887327/jobs/6356592666#step:6:15) tests in 0.0[18](https://github.com/AllenNeuralDynamics/aind-metadata-service/actions/runs/3743887327/jobs/6356592666#step:6:19)s

FAILED (failures=1)
Error: Process completed with exit code 1.

```

In the comparison, expected and actual results differ by `site-packages` vs `src`

**To Reproduce**
Steps to reproduce the behavior:
1. Create pull request
2. Wait for CI workflow to run
3. See error

**Expected behavior**
No error on test run

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -15,14 +15,14 @@ classifiers = [
 ]
 readme = "README.md"
 dynamic = ["version"]
-aind-data-schema = "0.7.4"
 
 dependencies = [
     'pyodbc',
     'office365-rest-python-client',
     'aind-data-schema',
     'fastapi',
-    'uvicorn[standard]'
+    'uvicorn[standard]',
+    'aind-data-schema'
 ]
 
 [project.optional-dependencies]

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -19,10 +19,9 @@ dynamic = ["version"]
 dependencies = [
     'pyodbc',
     'office365-rest-python-client',
-    'aind-data-schema',
+    'aind-data-schema==0.3.2',
     'fastapi',
-    'uvicorn[standard]',
-    'aind-data-schema'
+    'uvicorn[standard]'
 ]
 
 [project.optional-dependencies]

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -15,6 +15,7 @@ classifiers = [
 ]
 readme = "README.md"
 dynamic = ["version"]
+aind-data-schema = "0.7.4"
 
 dependencies = [
     'pyodbc',

--- a/src/aind_metadata_service/__init__.py
+++ b/src/aind_metadata_service/__init__.py
@@ -1,3 +1,3 @@
 """REST service to retrieve metadata from databases.
 """
-__version__ = "0.0.10"
+__version__ = "0.0.11"

--- a/tests/test_response_handle.py
+++ b/tests/test_response_handle.py
@@ -1,0 +1,62 @@
+""""Tests response_handler module"""
+
+import unittest
+
+from aind_data_schema.subject import Subject
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import JSONResponse
+from pydantic import validate_model
+
+from aind_metadata_service.response_handler import Responses
+
+
+class TestResponseHandler(unittest.TestCase):
+    """Tests methods in Responses class"""
+
+    def test_valid_model(self):
+        """Test model_response with valid model."""
+
+        model = Subject(
+            species="Mus musculus",
+            subject_id="639374",
+            sex="Male",
+            date_of_birth="2022-06-24",
+            genotype="Pvalb-IRES-Cre/wt;RCL-somBiPoles_mCerulean-WPRE/wt",
+        )
+        response = Responses.model_response(model)
+        model_json = jsonable_encoder(model)
+        expected_response = JSONResponse(
+            status_code=200,
+            content=(
+                {
+                    "message": "Valid Model.",
+                    "data": model_json,
+                }
+            ),
+        )
+
+        self.assertEqual(expected_response.status_code, response.status_code)
+        self.assertEqual(expected_response.body, response.body)
+
+    def test_invalid_model(self):
+        """Test model_response with invalid model."""
+        model = Subject.construct()
+        response = Responses.model_response(model)
+        *_, validation_error = validate_model(model.__class__, model.__dict__)
+        model_json = jsonable_encoder(model)
+        expected_response = JSONResponse(
+            status_code=418,
+            content=(
+                {
+                    "message": f"Validation Errors: {validation_error}",
+                    "data": model_json,
+                }
+            ),
+        )
+
+        self.assertEqual(expected_response.status_code, response.status_code)
+        self.assertEqual(expected_response.body, response.body)
+
+
+if __name__ == "__main__":
+    unittest.main()

